enrolment-ui/public/index.html

```diff
@@ -20,7 +20,6 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
This shouldn't be here, but bundled in the code so it can be processed and optimised by webpack.